cranelift/codegen/src/machinst/mod.rs

```diff
@@ -31,48 +27,33 @@
 //!         |                          with unknown offsets.
 //!         |                        - critical edges (actually all edges)
 //!         |                          are split.)
-//!         | [regalloc]
 //!         |
-//!     VCode<arch_backend::Inst>   (machine instructions:
-//!         |                        - all real registers.
-//!         |                        - new instruction sequence returned
-//!         |                          out-of-band in RegAllocResult.
-//!         |                        - instruction sequence has spills,
-//!         |                          reloads, and moves inserted.
-//!         |                        - other invariants same as above.)
-//!         |
-//!         | [preamble/postamble]
+//!         | [regalloc --> `regalloc2::Output`; VCode is unchanged]
```

Comment:
Will the disassembly now show virtual registers or still use physical registers?

Reply:
I see it still uses physical registers.